content/graphing/infrastructure/cloudfunctions.md

```diff
@@ -0,0 +1,134 @@
+
+	To enable this, refer to the [documentation for sending Lambda logs to Datadog][6].
+
+## Searching, Filtering, Sorting
+
+### Tagging
+
+Datadog tags your Lambda functions and metrics with relevant AWS metadata and custom tags.
+
+AWS Metadata:
+
+- account_id
+- executedversion
+- functionname
+- memorysize
+- region
+- resource
+- runtime
+
+Ensure you have the `lambda:List*` permission in your Datadog IAM policy in order to have these tags collected.
+
+Ensure you have the `tag:GetResources` permission in your Datadog IAM policy in order to have custom tags collected.
+
+### Filtering
+
+Use the faceted search functionality along the left-hand side of the page to narrow down the functions that are in view. All AWS and custom tags are available to use as filters.
+
+### Selecting metrics on table
+
+You can check and uncheck metrics you would like to view on the functions table. By default, invocations, duration (avg), and errors are on by default.
+
+Non-default metrics you can check are
+
+- ConcurrentExecutions
+- Throttles
+- IteratorAge
+- DeadLetterError
+- Estimated Cost
+- Memory Used (avg)
+- Last Start
+
+## Function Detail view
+
+Clicking on a particular function in the function summary table brings you to a function detail page. This page provides detailed trace and log level information for that function.
+
+{{< img src="graphing/infrastructure/cloudfunctions/cf-functiondetailview.png" alt="Cloud Functions - Function Detail View" responsive="true" style="width:80%;">}}
+
+### Summary graphs and time selector
+
+Use the summary graphs across the top of screen and the time selector to focus in on a specific time frame you are interested in. The entire page, including the traces and logs shown, updates when you use the time selector.
+
+### Traces
+
+Via the AWS X-Ray integration, traces from the function currently being viewed are shown in the `Traces` tab. You can sort these traces by attributes such as date, duration, and status.
+
+{{< img src="graphing/infrastructure/cloudfunctions/cf-traces.png" alt="Traces" responsive="true" style="width:80%;">}}
+
+#### Trace detail view
+
+Clicking on a particular trace opens the trace detail view for that trace. The X-Ray subsegments are transformed into Datadog spans while preserving the naming paradigms, span metadata, and structure of the overall trace.
+
+{{< img src="graphing/infrastructure/cloudfunctions/cf-traces2.png" alt="Traces" responsive="true" style="width:80%;">}}
+
+Datadog provides specially formatted serverless traces for readability and usability. Clicking on the span from another Lambda function creates a link to that function’s detail page so you can easily jump to another function that is part of the trace.
+
+{{< img src="graphing/infrastructure/cloudfunctions/cf-traces3.gif" alt="Traces" responsive="true" style="width:80%;">}}
```

Comment:
let's remove this. it's a bad one as the page it opens up is empty. If I make a new one in time i'll let you know